lib/src/views/feed/feed_screen.dart

```diff
@@ -0,0 +1,59 @@
+// ignore_for_file: prefer_const_constructors
+
+import 'package:feed_sx/src/data/local_db/local_db_impl.dart';
+import 'package:feed_sx/src/data/models/branding/branding.dart';
+import 'package:feed_sx/src/data/repositories/branding/branding_repository.dart';
+import 'package:feed_sx/src/sdk/branding_sdk.dart';
+import 'package:feed_sx/src/theme.dart';
+import 'package:feed_sx/src/views/feed/components/custom_app_bar.dart';
+import 'package:feed_sx/src/views/feed/components/post/post_actions.dart';
+import 'package:feed_sx/src/views/feed/components/post/post_description.dart';
+import 'package:feed_sx/src/views/feed/components/post/post_header.dart';
+import 'package:feed_sx/src/views/feed/components/post/post_image.dart';
+import 'package:feed_sx/src/views/feed/components/post/post_widget.dart';
+import 'package:feed_sx/src/widgets/loader.dart';
+import 'package:flutter/material.dart';
+import 'package:feed_sx/src/utils/constants/ui_constants.dart';
+
+class FeedScreen extends StatelessWidget {
```

Comment:
The screen has to be a stateful widget, since there are going to be changes to the UI. Why haven't you used the BLoC structure yet? All of the futures has to be done and data has to be passed from the BLoC using states and events.

Reply:
I decided not to create BLOCs right now because we don't have the ED for the SDK, and we don't know what models it will return or what methods will be exposed. Once it is done, I will start working on the BLOCs. It will be much more efficient and intuitive to implement the BLOCs after the ED.